### ISSUE TITLE
Broken Links

### ISSUE THREAD

**Issue by MartinMuzatko** [blitz-js/blitzjs.com#628]
Hi there!

I found a few broken links in the documentation. I think most of the missing `.md`  urls just come from automated documentation.
However, there are also a few urls to missing pages.

https://blitzjs.com/docs/get-static-paths links to https://blitzjs.com/docs/routing/dynamic-routes.md
https://blitzjs.com/docs/router links to https://blitzjs.com/docs/basic-features/data-fetching.md
https://blitzjs.com/docs/router#router-object links to https://blitzjs.com/docs/advanced-features/preview-mode.md

https://blitzjs.com/docs/pages#persisting-layouts links to https://blitzjs.com/docs/layout
https://blitzjs.com/docs/mutation-usage links to https://blitzjs.com/docs/resolver-utilities (see `invalidateQuery()`)
https://blitzjs.com/docs/blitz-config#disable-keep-alive links to https://blitzjs.com/docs/basic-features/supported-browsers-features#polyfills

**Comment by beerose** [comment]
Thanks for creating the issue!

**Comment by corydeppen** [comment]
The persisting layouts link is the only one that appears to be broken at the moment, but I was wondering whether some of these are still valid based on the [documentation.json](https://github.com/blitz-js/blitzjs.com/blob/main/app/core/navs/documentation.json) file. Are files in `/pages/docs` valid if they're not in that file, or does that file need to be updated to account for files in the docs directory that may not be in the nav?

**Comment by beerose** [comment]
@corydeppen, yeah, we removed a bunch of pages from our docs after moving to Blitz 2.0. They were redundant as it was mostly the same stuff as you can find in the official Next.js docs. Though, we didn't remove the markdown files from the repo (which is something we should do soon)

**Comment by ricky0123** [comment]
There are some broken URI fragments in the "Notes" box in the [session management](https://blitzjs.com/docs/session-management), ie

* https://blitzjs.com/docs/session-management#sessioncontext
* https://blitzjs.com/docs/session-management#access-session-on-the-server

There are no correspoding elements on the page.

**Comment by VedanthB** [comment]
I would like to contribute to this ?
